Modules/timemodule.c

```diff
@@ -622,27 +656,27 @@ time_strftime(PyObject *self, PyObject *args)
     fmt = PyBytes_AS_STRING(format);
 #endif
 
-#if defined(MS_WINDOWS) && !defined(HAVE_WCSFTIME)
+#if defined(MS_WINDOWS) && defined(HAVE_WCSFTIME)
```

Comment:
What abut the case `defined(MS_WINDOWS) && !defined(HAVE_WCSFTIME)`?

Reply:
If I'm not mistaken, this case is possible only when we undefine HAVE_WCSFTIME somewhere. Do you mean we should have special code for this case?
Now I just returned this block to state before switching from wcsftime to strftime.